src/getCommitMeaning.ts

```diff
@@ -9,7 +9,14 @@ const releaseCommitTester =
 
 export function getCommitMeaning(message: string) {
 	// Some types are always meaningful or ignored, regardless of potentially release-like messages
-	const { type } = conventionalCommitsParser.sync(message);
+	const { notes, type } = conventionalCommitsParser.sync(message, {
+		// @ts-expect-error - options from https://github.com/conventional-changelog/conventional-changelog/issues/648#issuecomment-704867077
+		breakingHeaderPattern: /^(\w*)(?:\((.*)\))?!: (.*)$/,
+	});
```

Comment:
[Refactor] No need for a `// @ts-expect-error` when an assertion can do:

```suggestion
	// options from https://github.com/conventional-changelog/conventional-changelog/issues/648#issuecomment-704867077
	const { notes, type } = conventionalCommitsParser.sync(message, {
		breakingHeaderPattern: /^(\w*)(?:\((.*)\))?!: (.*)$/,
	} as object);
```

This is actually a great opportunity to clean up the DefinitelyTyped types for `conventional-commits-parser`. Is that something you have time for too? No worries if not, I will if you don't. 🙂

Reply:
I'll try to deal with it. It’s always exciting to have the opportunity to work on an open source project.